### DISCUSSION TITLE
关于存档迁移的问题

### DISCUSSION THREAD

**Discussion by peacefgf** [Meloong-Git/PCL#4470] [提问] [answered]
你好，我想问一下如果我想把一个存档移到更高的版本中，例如新版本草原中会生成一个新的物种，那我的旧存档移动后，出生点所在地的草原会不会刷新并生成这种新版本的新物种，谢谢！

**Comment by allMagicNB** [comment]
反正你去过的地方的地形不会变。

**Comment by LTCatt** [comment, marked as answer]
草原地形不会刷新，但会出现新的生物
